content/docs/1.17/operator.md

```diff
@@ -1013,13 +1013,48 @@ metadata:
   name: jaeger
 ```
 
+The Jaeger Operator also provides extensive logging when the flag `--log-level` is set to `debug`. Here's an excerpt from an `operator.yaml` that has the logging level set to debug:
+
+```yaml
+        # .Spec.Template.Spec.Containers[0].Args
+        args: ["start", "--log-level=debug"]
+```
+
+Note that tracing and logging at debug level can be both enabled at the same time.
+
 {{< info >}}
-Currently the Operator Lifecycle Manager (OLM) does not offer a way to configure an operator. As a result, if you install the Jaeger Operator via OLM, collecting traces from the Operator is not supported.
+Currently the Operator Lifecycle Manager (OLM) does not offer a way to configure an operator. As a result, if you install the Jaeger Operator via OLM, collecting traces or changing the log level from the Operator is not supported.
 {{< /info >}}
 
 # Monitoring the operator
 
-The Jaeger Operator starts a Prometheus-compatible endpoint on `0.0.0.0:8383/metrics` with internal metrics that can be used to monitor the process.
+The Jaeger Operator starts a Prometheus-compatible endpoint on `0.0.0.0:8383/metrics` with internal metrics that can be used to monitor the process. Interesting metrics to watch are:
+
+```
+# Total number of reconciliations and their outcomes (cumulative)
+controller_runtime_reconcile_total{controller="jaeger-controller",result="error"}
+controller_runtime_reconcile_total{controller="jaeger-controller",result="success"}
+
+# Total number of retries (cumulative)
+workqueue_retries_total{name="jaeger-controller"}
+
+# Current number of reconciliation loops in progress (gauge)
+workqueue_depth{name="jaeger-controller"}
+
+# How many seconds of work has done that is in progress and hasn’t been observed by work_duration. (gauge)
```

Comment:
Is it possible to express this is a clearer way? Feel free to explain at length and I can try to provide a more concise definition.

Reply:
Absolutely! This is the default text that people see from `/metrics`, but this doc should really explain it in our context.

Reply:
How about this? 

> # How long (in seconds) the oldest reconciliation loop is taking. (gauge)

When there is only one reconciliation loop, this gauge shows how long the reconciliation is taking. When there are multiple, it shows the value for the oldest reconciliation loop (ie, the one that is taking the longest).